VisualizationBase/src/items/VInfoNode.cpp

```diff
@@ -28,6 +28,9 @@
 #include "declarative/DeclarativeItemDef.h"
 #include "nodes/InfoNode.h"
 #include "Comments/src/items/VCommentBrowser.h"
+#include "utils/InfoJavascriptFunctions.h"
+#include "QtWebKitWidgets/QGraphicsWebView"
```

Comment:
I really don't want to have a dependency on WebKit in VisualizationBase. Please remove these includes. I'll comment below how to avoid needing them.